figures/3.figure3/scripts/3.assemble_montage.r

```diff
@@ -141,159 +149,146 @@ apoptosis_pyroptosis$channel_learned <- dplyr::recode(apoptosis_pyroptosis$chann
             .missing="other"
     )
 
-
-# change the text in the dfs
-control_pyroptosis$comparison <- "Control vs. Pyroptosis"
-apoptosis_control$comparison <- "Apoptosis vs. Control"
-apoptosis_pyroptosis$comparison <- "Apoptosis vs. Pyroptosis"
+print(dim(control))
+print(dim(apoptosis))
+print(dim(pyroptosis))
 
 # select rows to keep in the df by index
-rows_to_keep = c(5,10,12)
-control_pyroptosis = control_pyroptosis[rows_to_keep,]
+# rows_to_keep = c(1,8,15)
+# control = control[rows_to_keep,]
 
 # loop through the rows of the df and plot the images
-list_of_plots_control_pyroptosis <- c()
-for (i in 1:nrow(control_pyroptosis)){
-    list_of_plots_control_pyroptosis[[i]] <- get_image(control_pyroptosis, i)
+list_of_plots_control_plot <- c()
+for (i in 1:nrow(control)){
+    list_of_plots_control_plot[[i]] <- get_image(control, i)
 }
 
 # get the subtitle from each item in the list
-ggplot_objects_control_pyroptosis <- Map(
+ggplot_objects_control <- Map(
     add_title,
-    plot = list_of_plots_control_pyroptosis,
+    plot = list_of_plots_control_plot,
     title = (
-        paste0(control_pyroptosis$comparison, "\n",
-        control_pyroptosis$group, "\n",
-        control_pyroptosis$compartment, "\n",
-        control_pyroptosis$feature_group, "\n",
-        control_pyroptosis$measurement, "\n",
-        control_pyroptosis$channel_cleaned
+        paste0(control$comparison, "\n",
+        control$group, "\n",
+        control$compartment, "\n",
+        control$feature_group, "\n",
+        control$measurement, "\n",
+        control$channel_cleaned
 
         )))
+length(ggplot_objects_control)
```

Comment:
there are a few examples of this, where debugging items are interspersed throughout. This, to an external viewer, looks unprofessional. If it is indeed important to know the length of this variable, I recommend adding a phrase describing why (you don't need these for standard checks, like `df.head()`)